### PR TITLE
dcsa tpdoc example added

### PR DESCRIPTION
**Pull request by nissimsan**
https://github.com/w3c-ccg/traceability-vocab/issues/242

### CODE REVIEW

File: docs/openapi/components/schemas/common/DCSATransportDocument.yml [hunk 1]
```diff
@@ -186,5 +186,169 @@ required:
...
+            "addressLocality": "Kowloon Bay",
+            "addressRegion": "Hong Kong",
+            "addressCountry": "Hong Kong SAR"
+          }
+        },
+        "vesselNumber": "HMM Algeciras",
+        "voyageNumber": "V.0004W"
+      }
+    },
+    "@context": [
```
Comment (OR13) [inline docs/openapi/components/schemas/common/DCSATransportDocument.yml:343]: I kinda don't like the sort order of these fields, prefer this at the top

File: docs/openapi/components/schemas/common/DCSATransportDocument.yml [hunk 2]
```diff
@@ -186,5 +186,169 @@ required:
...
+          "phoneNumber": "+32-3-201.98.10"
+        },
+        "cargoItems": [
+          {
+            "type": "CargoItem",
+            "cargoLineItems": [
+              {
+                "type": "CargoLineItem",
+                "descriptionOfGoods": "Rollators",
+                "shippingMarks": "ByAcre Age is the new Black"
```
Comment (OR13) [inline docs/openapi/components/schemas/common/DCSATransportDocument.yml:266]: wut is this saying

File: docs/openapi/components/schemas/common/DCSATransportDocument.yml [hunk 3]
```diff
@@ -186,5 +186,169 @@ required:
   - shippingInstruction
 example: |-
   {
-    "issue": "https://github.com/w3c-ccg/traceability-vocab/issues/242"
+    "id": "http://example.org/credentials/",
```
Comment (OR13) [inline docs/openapi/components/schemas/common/DCSATransportDocument.yml:189]: is is sad, lacks identifier

**Review by OR13 — APPROVED** [review]

**Review by mprorock — CHANGES_REQUESTED** [review]
needs an identifier

**Comment by OR13** [comment]
ideally a unique identifier XD

File: docs/openapi/components/schemas/common/DCSATransportDocument.yml (hunk 1 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/DCSATransportDocument.yml:343]: For sure, completely agree.

File: docs/openapi/components/schemas/common/DCSATransportDocument.yml (hunk 2 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/DCSATransportDocument.yml]: image
"Marks and numbers". It's what's printed on the boxes, assisting consignee/customs/others in determining and organizing what's in the container.

**Comment by nissimsan** [comment]
Closing, replaced by https://github.com/w3c-ccg/traceability-vocab/pull/266

**Pull request closed without merging**